### DISCUSSION TITLE
How to use message application command in private dms (of the user)

### DISCUSSION THREAD

**Discussion by Pixel2075** [Pycord-Development/pycord#2478] [Q&A] [answered]
I want to use message application command (context menu commands) in user's private dms with other users than the bot itself. I have user-installed the bot.

```
    @commands.message_command(
        name="Remind Me",

    )
    async def message_com(self, ctx, message):
        view = util.reminderInteraction.ReminderSelectView(
            self.process_reminder,
            ctx=ctx,
            message_content=message.content,
            message_by=message.author.global_name,
        )
        await ctx.respond(view=view, ephemeral=True)

```

This remind me only shows up in the guilds or the bot's dm.
![image](https://github.com/Pycord-Development/pycord/assets/99288538/f1df33c7-b9ca-4e71-8725-b00fce6673aa)

I want to use the message command in private dms with other users.

**Comment by plun1331** [comment, marked as answer]
#2409
